iati/core/tests/test_schemas.py

```diff
@@ -188,10 +204,18 @@ def test_schema_rulesets_add(self, schema_initialised):
 
     @pytest.mark.skip(reason='Not implemented')
     def test_schema_rulesets_add_twice(self, schema_initialised):
-        """Check that it is not possible to add the sameRulesets to a Schema multiple times."""
+        """Check that it is not possible to add the sameRulesets to a Schema multiple times.
+
+        Todo:
+            Consider if this test should test against a versioned Ruleset.
```

Comment:
Do we have other Ruleset versions?

Reply:
From a technical perspective, a Ruleset should be deemed as valid against a particular Decimal Version of the Standard. That they are all the same is, well, IATI for you.